compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -139,6 +139,24 @@ internal class ComposeContainer(
         composeSceneFactory = ::createComposeScene,
     )
 
+    /**
+     * The listener to [windowContainer] size and position changes.
+     */
+    private val windowContainerComponentListener = object : ComponentAdapter() {
+        override fun componentResized(e: ComponentEvent?) = onWindowContainerSizeChanged()
+        override fun componentMoved(e: ComponentEvent?) = onWindowContainerPositionChanged()
+    }
+
+    /**
+     * The listener to [window] size changes.
+     */
+    private val windowSizeListener = object : ComponentAdapter() {
+        // When the window is moved to a display with a different density, componentResized is
+        // called on the window, but not on `windowContainer`, so we need to update the size
+        // here too.
```

Comment:
Is it possible to have a test for this case?

Reply:
I can't think of an easy way.